compaction/compaction_manager.hh

```diff
@@ -423,6 +423,8 @@ public:
     bool requires_cleanup(table_state& t, const sstables::shared_sstable& sst) const;
     const std::unordered_set<sstables::shared_sstable>& sstables_requiring_cleanup(table_state& t) const;
 
+    future<> on_compaction_completion(table_state& t, sstables::compaction_completion_desc desc, sstables::offstrategy offstrategy);
+
```

Comment:
private? task executors are friend classes.

Reply:
made private in next version